plugins/chat/assets/javascripts/discourse/models/chat-thread.js

```diff
@@ -31,6 +31,7 @@ export default class ChatThread {
   @tracked tracking;
   @tracked currentUserMembership;
   @tracked preview;
+  @tracked force;
```

Comment:
Does this need to be a tracked property?

Reply:
You are right that it's probably not necessary ATM, but for now I think I prefer to consider every public model properties as tracked. It's so unexpected to have something untracked when working with a model